### PR TITLE
Update docs to specify where PeerManager::process_events is called

### PR DESCRIPTION
**Pull request by vss96**
Doc changes for #1075

### CODE REVIEW

**Comment by vss96** [comment]
@TheBlueMatt I hope this should be enough to update the docs?

File: lightning/src/ln/peer_handler.rs [hunk 1]
```diff
@@ -1165,10 +1165,13 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	/// Checks for any events generated by our handlers and processes them. Includes sending most
 	/// response messages as well as messages generated by calls to handler functions directly (eg
 	/// functions like [`ChannelManager::process_pending_htlc_forwards`] or [`send_payment`]).
-	///
+	///
```
Comment (TheBlueMatt) [inline lightning/src/ln/peer_handler.rs:1168]: nit: excess whitespace?

File: lightning/src/ln/peer_handler.rs [hunk 2]
```diff
@@ -1165,10 +1165,13 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	/// Checks for any events generated by our handlers and processes them. Includes sending most
 	/// response messages as well as messages generated by calls to handler functions directly (eg
 	/// functions like [`ChannelManager::process_pending_htlc_forwards`] or [`send_payment`]).
-	///
+	/// 
 	/// May call [`send_data`] on [`SocketDescriptor`]s. Thus, be very careful with reentrancy
 	/// issues!
 	///
+	/// You don't have to call this function explicitly if you are using lightning-net-tokio
```
Comment (TheBlueMatt) [inline lightning/src/ln/peer_handler.rs:1172]: nit: lightning-net-tokio should be surrounded by `s.

**Review by TheBlueMatt — COMMENTED** [review]
I think we should mention this call in lightning-net-tokio's three public methods that create connections, but this is fine either way.

**Comment by TheBlueMatt** [comment]
Can you squash the commits into one? Then happy to land this.

**Comment by vss96** [comment]
I squashed it into a single commit, but for some reason the check_commits stage is failing

**Comment by TheBlueMatt** [comment]
According to github, you now have four commits in the PR. See https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits for instructions on how to squash and force-push.

**Comment by vss96** [comment]
@valentinewallace I'm not sure how your changes got in when I pushed I could only see my changes in the diff.

**Comment by TheBlueMatt** [comment]
Huh! I'm not entirely sure how that happened. In any case, I went ahead and took the previous version of this PR from yesterday, squashed the commit into one, and pushed it (github, by default, allows maintainers to push to your branches that are associated with a PR).

**Review by TheBlueMatt — APPROVED** [review]

**Review by valentinewallace — APPROVED** [review]

**Comment by vss96** [comment]
Thank you for the help! :)

**Pull request merged**